storage_service/locations/forms.py

```diff
@@ -103,14 +103,6 @@ class Meta:
         model = models.Dataverse
         fields = ('host', 'api_key', 'agent_name', 'agent_type', 'agent_identifier')
 
-    def as_p(self):
-        # Add a warning to the Dataverse-specific section of the form
-        # FIXME this may not be the best way to add Space-specific warnings
-        content = super(DataverseForm, self).as_p()
-        content += '\n<div class="alert">{}</div>'.format(
-            _('Integration with Dataverse is currently a beta feature'))
-        return content
```

Comment:
Maybe this is fixable by using [`mark_safe`](django.utils.safestring.mark_safe) again?

Reply:
`mark_safe()` didn't come up in the literature I was reading as part of this PR but having tested it - it looks to work. I agree this is the right approach. The `(Beta)` component of this work is temporary so a migration is not ideal.